2d/pong/godot/ball.gd

```diff
@@ -0,0 +1,23 @@
+extends KinematicBody2D
+
+var MAX_VELOCITY = 300
+var velocity = Vect	or2.ZERO
+
+
+func _ready():
+	#Initial velocity.
+	randomize() 
```

Comment:
I propose use docstrings like this:
```gdscript
func foo(arg)
    """
    Initial velocity

    :param arg: Argument...
    :return: Randomized argument.
    """
    bar = arg * randomize()
    return bar
```
instead of using `# comments`. I think is better use only the essential comments.

(This is only an example, because this concrete function doesn't needs such docstring, of course)